Makefile

```diff
@@ -31,6 +31,19 @@ DIST = $(DISTDIR)/.done
 GIT_HEAD_HASH = $(shell git rev-parse HEAD)
 BUILDBRANCH ?= staging
 
+# "Baked in" environment variables
+# for built JS bundle
+NODE_ENV = production
+GITHUB_API_ENDPOINT = https://api.github.com
+STORE_DEVPORTAL_URL = https://myapps.developer.ubuntu.com/dev
+
+target staging:
+	BASE_URL = https://build.staging.snapcraft.io
+
+target production:
+	BASE_URL = https://build.snapcraft.io
+
```

Comment:
☝️ This is my random, uninformed stab at setting the environment variables we need in Jenkins when we build the JS bundle. Some thoughts:

1) Is there a better place for these settings to "live"?
2) Have I used the correct syntax for setting variables and switching targets?
3) Do we already have staging and production targets set up on Jenkins?